$NetBSD: patch-dfasyn_Makefile,v 1.2 2020/09/26 14:25:52 mef Exp $

Fix MAKE_JOBS build; without this hack it runs two concurrent copies
of bison that produce garbage output.

2. fix for bison-3.7.1

--- dfasyn/Makefile.orig	2007-06-22 20:18:00.000000000 +0000
+++ dfasyn/Makefile
@@ -47,8 +47,8 @@ dfasyn : $(OBJ)
 
 parse.c parse.h : parse.y
 	bison -v -d parse.y
-	mv parse.tab.c parse.c
-	mv parse.tab.h parse.h
+	cp -f parse.tab.c parse.c
+	cp -f parse.tab.h parse.h
 
 parse.o : parse.c dfasyn.h
 
